docs/MO_DG/prepare_model/convert_model/supported_model_formats.md

```diff
@@ -16,32 +16,551 @@
    openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_Kaldi
    openvino_docs_MO_DG_prepare_model_convert_model_tutorials
 
-.. meta::
-   :description: In OpenVINO, ONNX, PaddlePaddle, TensorFlow and TensorFlow Lite 
-                 models do not require any prior conversion, while MxNet, Caffe and Kaldi do.
 
+**OpenVINO IR (Intermediate Representation)** - the proprietary and default format of OpenVINO, benefiting from the full extent of its features. All other model formats presented below will ultimately be converted to :doc:`OpenVINO IR <openvino_ir>` either manually or automatically.
 
-**OpenVINO IR (Intermediate Representation)** - the proprietary format of OpenVINO™, benefiting from the full extent of its features.
+**ONNX, Pytorch, PaddlePaddle, TensorFlow, and TensorFlow Lite** are supported directly, which means they may be used without any prior conversion and can be read directly by OpenVINO Runtime API by the use of ``read_model()`` or ``compile_model()``. Additional adjustment for the model can be performed using the ``convert_model()`` method, which allows you to set shapes, types or the layout of model inputs, cut parts of the model, freeze inputs etc. The detailed information of capabilities of  ``convert_model()`` can be found in :doc:`this <openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide>` article.
```

Comment:
The description is correct for ovc.convert_model(), except for the statement that it allows to set layout, cut model or freeze inputs.
@slyalin do we need to update the documentation to ovc.convert_model() model? Do we need to remove articles about layout or input/output cut?

Reply:
Commented on `directly` and `indirectly` supported. Please avoid using these separation. Describe specific APIs and their availability in different circumstances (Python vs C++ vs CLI, model is represented as a file/a Python object etc.) instead of replacing by `directly`/`indirectly`.

Reply:
From the Python user perspective there is no real difference in availability of convert_model, read_model and compile_model that differentiate them as `direct` or `indirect` support. From the user perspective they all are the direct support. It is better to start with user scenarios and recommend the proper way to follow depending on the scenario.

Reply:
I have applied the changes with the https://github.com/openvinotoolkit/openvino/pull/18495/commits/5e2c5e1914622cc0a9d7b5e1ec002ecaf9378b75 commit.